examples/postgres.yaml

```diff
@@ -30,5 +30,5 @@ metadata:
   name: myappschema
 spec:
   database: myappdb
-  git: github://<GITHUB USERNAME>:<GITHUB PASSWORD>@infobloxopen/atlas-contacts-app/db/migrations
+  source: github://<GITHUB USERNAME>:<GITHUB PASSWORD>@infobloxopen/atlas-contacts-app/db/migrations
```

Comment:
give an example for s3 as source as well.

Reply:
new yaml file ?